dll/win32/kernel32/winnls/string/lang.c

```diff
@@ -2014,27 +2014,29 @@ INT WINAPI LCMapStringEx(LPCWSTR name, DWORD flags, LPCWSTR src, INT srclen, LPW
         INT convlen = dst_ptr - dst;
         for (dst_ptr = dst; convlen; --convlen, ++dst_ptr)
         {
+            /*
+             * U+3041 ... U+3093: Hiragana
+             * U+3095: HIRAGANA LETTER SMALL KA
```

Comment:
Why written in all caps by the way? :thinking: :joy:

Reply:
Fixed.